addons/dragging/initKeybinds.sqf

```diff
@@ -0,0 +1,58 @@
+// Add Keybinds
+["ACE3 Common", QGVAR(drag), LLSTRING(DragKeybind), {
+    private _player = ACE_player;
+
+    if (!alive _player) exitWith {false};
+    if !([_player, objNull, ["isNotDragging", "isNotCarrying"]] call EFUNC(common,canInteractWith)) exitWith {false};
```

Comment:
As far as I can tell, you are meant to be able to drag while swimming.
I've never quite understood the logic of exclusions of `EFUNC(common,canInteractWith)` - Does this account for being able to drag while swimming?

Reply:
```suggestion
    if !([_player, objNull, ["isNotDragging", "isNotCarrying", "isNotSwimming"]] call EFUNC(common,canInteractWith)) exitWith {false};
```